src/SlowSlothBudget.Web/Startup.cs

```diff
@@ -58,9 +60,15 @@ public void ConfigureServices(IServiceCollection services)
                     {
                         options.Authority = domain;
                         options.Audience = Configuration["Auth0:ClientID"];
+
+                        //BART: maybe it would work? So you dont need to validate in requirement
+                        //options.TokenValidationParameters.ValidateIssuer = true;
+                        //options.TokenValidationParameters.ValidIssuer = "your issuer?";
                     });
             }
 
+            // BART: any reason why one repository class is transient and one is singleton?
+
```

Comment:
Good question. I don't recall any particular reason, so it's probably a mistake. Not mentioning MongoDB client usage in this project is a mess. I have tasks to improve it in my backlog ;)